apps/web/src/pages/_app.tsx

```diff
@@ -3,13 +3,45 @@ import '../styles.css';
 import Loading from '@components/Shared/Loading';
 import circluarStd from '@lib/lensterFont';
 import type { AppProps } from 'next/app';
-import { lazy, Suspense } from 'react';
+import Head from 'next/head';
+import { lazy, Suspense, useMemo } from 'react';
+import { useAppPersistStore } from 'src/store/app';
 
 const Providers = lazy(() => import('@components/Common/Providers'));
 
 const App = ({ Component, pageProps }: AppProps) => {
+  const notificationCount = useAppPersistStore(
```

Comment:
We updated the notification persist store

Reply:
Gotcha. Created a new attribute `hasUnreadNotifications` in store to get access to it from `Favicon.tsx`